Sensors/Camera/Camera.java

```diff
@@ -0,0 +1,52 @@
+package POPLib.Sensors.Camera;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.photonvision.EstimatedRobotPose;
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonPoseEstimator;
+import org.photonvision.PhotonPoseEstimator.PoseStrategy;
+import org.photonvision.targeting.PhotonPipelineResult;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.Pose2d;
+
+public class Camera {
+    private final PhotonCamera camera;
+    private final CameraConfig config;
+    private PhotonPoseEstimator poseEstimator;
+
+    public Camera(CameraConfig config) {
+        this.config = config;
+        camera = new PhotonCamera(config.cameraName);
+        AprilTagFieldLayout aprilTags = AprilTagFieldLayout.loadField(AprilTagFields.k2024Crescendo);
```

Comment:
Make year configuration a parameter in constructor so class can be reused without changing lib from year to year